extensions/wikia/Wall/templates/Wall_message.php

```diff
@@ -122,7 +122,11 @@
 					<? $i++; ?>
 				<? endforeach; ?>
 			<? endif; ?>
-			<?= $app->renderViewCached( 'WallController', 'reply', 'Wall_message'.$showReplyForm, array('showReplyForm' => $showReplyForm )); ?>
+			<? if( $repliesNumber < $repliesLimit ) {
+				echo $app->renderViewCached( 'WallController', 'reply', 'Wall_message'.$showReplyForm, array('showReplyForm' => $showReplyForm ));
+			} else {
+				echo "<div class=message-topic-error >" . wfMsgExt('wall-message-limit-reached', array('parsemag'), array( $repliesLimit ));
```

Comment:
Oh, it's missing.  DIdn't affect the rendering/output so I didn't notice.  Will fix.